### ISSUE TITLE
puic4 -x does not create valid perl code

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrisburel/perlqt4#44]
```
What steps will reproduce the problem?
1. create designer x.ui file.
2. puic4 -x -o Ui_MainWindow.pm qt_mainform.ui
3. perl Ui_MainWindow.pm

What is the expected output?
The form created in designer should be shown.

What do you see instead?
Perl errors arising from the missing semi-colons in the subroutine
created due to the -x option.
In addition $u is not ever used in this case.

sub main {
    my $a = Qt::Application(\@ARGV)
    my $w = Qt::MainWindow
    my $u = Ui_MainWindow
    $w->show()
    $a->exec()
}

What version of PerlQt4 are you using? 
  4:4.7.1-0ubuntu1   

What version of Qt are you using? On
  4:4.7.4-0ubuntu8.1

what operating system?
  ubuntu 11.10


Please provide any additional information below.

```

Original issue reported on code.google.com by `doug.bre...@gmail.com` on 15 Feb 2012 at 11:38

**Comment by chrisburel** [comment]
Fixed in 0a417a63b428a0a577cdb2794cb90b4a4015567a.

**Comment by chrisburel** [comment]
Note that I applied Olivier's patch as it was, without changing any of the inheritance of the generated classes as mentioned in comment 4.
